src/interpreter.ts

```diff
@@ -638,40 +638,16 @@ export class Interpreter<
     }
   }
   private defer(sendAction: SendActionObject<TContext, TEvent>): void {
-    let { delay } = sendAction;
-
-    if (isString(delay)) {
```

Comment:
sends are already being resolved in StateNode so I couldn't find a reason to support this resolving in Interpreter too + no test has failed because of removing this